app/src/main/java/org/fossasia/openevent/app/core/event/list/EventsViewModel.java

```diff
@@ -73,4 +85,26 @@ public void loadUserEvents(boolean forceReload) {
         return success;
     }
 
+    public void filter() {
+        List<Event> live = new ArrayList<>();
+        List<Event> past = new ArrayList<>();
+        List<Event> draft = new ArrayList<>();
+
+        for (Event event : events.getValue()) {
+            try {
+                if (event.getState().equalsIgnoreCase("draft"))
```

Comment:
The server accepts only lowercase value for event state. So, case shouldn't be ignored while comparing.